pkg/cmd/cli/restore/create.go

```diff
@@ -165,15 +165,10 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 		return errors.New("Velero client is not set; unable to proceed")
 	}
 
-	switch {
-	case o.BackupName != "":
+	if o.BackupName != "" {
```

Comment:
Referring to the checks on L#151 and L#155, should we validate `ScheduleName` at all?